com.plexamp.Plexamp.yaml

```diff
@@ -55,3 +59,13 @@ modules:
           is-main-source: true
           type: electron-updater
           url: https://plexamp.plex.tv/plexamp.plex.tv/desktop/latest-linux.yml
+    cleanup:
+      # Config for the self-updater?
+      # Manages to crash something, but the flatpak can't self-update, anyway.
+      - app-update.yml
+      - cacert.pem
```

Comment:
We can't remove this cert. AFAIK it's used for more than just the updater. Off the top of my head, I believe it's required for relayed server connections.